std/log/logger.ts

```diff
@@ -37,86 +36,67 @@ export class LogRecord {
   }
 }
 
-export interface LoggerOptions {
-  handlers?: BaseHandler[];
-}
+const DEFAULT_LEVEL = "Debug";
+const DEFAULT_LOGGER_NAME = "logger";
+const defaultConsoleHandler = new ConsoleHandler(DEFAULT_LEVEL);
+const DEFAULT_HANDLERS = [defaultConsoleHandler];
```

Comment:
Please revert the default to `DEFAULT_HANDLERS = []`. Otherwise you can not create a silent logger.

One may want to do this, if one wants to log conditionally, for example if the program is run in certain environments. One can not just not initialise a logger, because all the logging statements in the code would then throw.

Reply:
Hei @vwkd  thanks for your feedback. I changed the default log level to debug because imo the logger Should function similar to console by default, logging everything by default and adjust that by setting a specific log level.

Concerning the default handler: to create a silent logger, you can pass an empty array as handlers. I think this is much more clear instead of having the default logger do nothing.

Reply:
Thanks @timreichen passing an empty array to `new Logger({ handlers: [] })` actually works. Consider my previous comment as obsolete.

Regarding the default log level from my other comment: A logger framework is not intended to be similar to `console` or trying to replace it, because it has a entirely different use case. Logging for real time inspection (with `console`) and logging for retrospective inspection (with a logging library) are two very different things. It just so happens that a logging library usually also has a output channel to the console, but that is only one out of many, like for example file based logging. In no other language you'll find that the logging framework aligns to the console output, e.g. `print` in python. I'm not in favor of aligning the logging library with `console` because it has an entirely different use case.

Reply:
@vwkd thanks for the explanations.
### on console alignment
Maybe I did not explain well enough what I meant with `console` alignment. This module should definitely not be a replacement to `console`.
I think std modules are supposed to provide certain functionality as bear bone as possible without or the minimal possible opinion so it does not become a framework on its own.
Since there are so many different logging frameworks that not or only partially agree on log levels, namings etc. I took `console` as a guide for it is a standard web API.

### on python implementation
On research there are many frameworks that have different [log level defaults](https://github.com/denoland/deno/pull/7293#discussion_r506958120) and have  no `critical` log level at all or call it `fatal` instead of `critical`. I do not see why we should follow pythons implementation of logger other than it was the predecessor of this PR.  Or are there any specific reasons for it?
_Side note_: I would agree on implementing `trace` as a log level, but strictly speaking it is not a python log level.

### on default log level
Since different language/frameworks have different default log levels a easy and non-opinionated solution would be to not have a default one at all but request the  a specific log level in instance initialization instead.
### on default handler
I might agree on that one. Maybe it is not right to have consoleHandler as a default handler. If there was no default log level, it would be consistent not to have a default handler too.

What would you think of adjusting the `Logger` class to be simple to extend with custom log levels?